common/config.js

```diff
@@ -59,7 +60,10 @@
                 pid: MathUtils.uuid(),
                 wid: $window.name
             },
-            hideNavbar: hideNavbar
+            settings: {
```

Comment:
Might make sense to store all the "settings" in this object, even if they aren't consumed beyond the config app setup and setupHead function.